home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -2210,9 +2251,11 @@ components:
                   - id: 3
                     type: email
                     main: true
+                    email: "get2.example@mail.com"
                   - id: 4
                     type: phone
                     main: false
+                    phone: "+380506617306"
               ownership_part: 0.38
 
```

Comment:
Is it necessary that in the example the sum of the ownership shares does not exceed 1?

Reply:
> Is it necessary that in the example the sum of the ownership shares does not exceed 1?

this question is beyond the scope of my assignment, it is better to ask it during a meeting with a technical expert and / or business analysts

Reply:
@IhorSamoshost Yes, we have such validation.